georiviere/valorization/registry.py

```diff
@@ -0,0 +1,8 @@
+from django.utils.translation import gettext_lazy as _
+from mapentity.registry import MapEntityOptions
+
+
+class ValorizationOptions(MapEntityOptions):
+    def __init__(self, model):
+        super().__init__(model)
+        self.label = _('Valorization')
```

Comment:
why don't just set label attribute in class ?

Reply:
The label attribute is set again in the init of mapentityoptions